runtime/moonriver/tests/integration_test.rs

```diff
@@ -1405,7 +1405,7 @@ fn min_gas_fee_is_correct() {
 
 		assert_eq!(
 			TransactionPaymentAsGasPrice::min_gas_price(),
-			(312_500_000u128.into(), Weight::from_parts(41_742_000u64, 0))
+			(312_500_000u128.into(), Weight::from_parts(59_217_000u64, 0))
```

Comment:
This is the read cost, we should use `<Runtime as frame_system>::DbWeight::get().read` or `moonbase_weights::db::rocksdb::constants::RocksDbWeight::get().read`.